apps/dashboard/src/components/workflow-editor/steps/in-app-editor.tsx

```diff
@@ -0,0 +1,113 @@
+import { RiEdit2Line, RiInformationFill, RiPencilRuler2Line } from 'react-icons/ri';
+import { Cross2Icon } from '@radix-ui/react-icons';
+import { useNavigate } from 'react-router-dom';
+import { useFormContext } from 'react-hook-form';
+import * as z from 'zod';
+
+import { Button } from '@/components/primitives/button';
+import { Separator } from '@/components/primitives/separator';
+import { SheetClose } from '@/components/primitives/sheet';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/primitives/tabs';
+import { Notification5Fill } from '@/components/icons';
+import { AvatarPicker } from '@/components/primitives/form/avatar-picker';
+import { Input, InputField } from '@/components/primitives/input';
+import { Textarea } from '@/components/primitives/textarea';
+import { workflowSchema } from '../schema';
+import { ActionPicker } from '../action-picker';
+import { URLInput } from '@/components/primitives/url-input';
+import { urlTargetTypes } from '@/utils/url';
+import { RedirectTargetEnum } from '@novu/shared';
+
+const tabsContentClassName = '!mt-0 h-full w-full px-3 py-3.5';
+
+export const InAppEditor = () => {
```

Comment:
in-app editor specific content, a lot of things is still missing:
- CodeMirror editors
- no schema fetching; the schema generated UI form
- the fields are not connected with react-hook-form